iam_groups_authn/sync.py

```diff
@@ -51,21 +53,34 @@ def get_group_members(self, group):
             members: List of all members (groups or users) that belong to the IAM group.
         """
         # build service to call Admin SDK Directory API
-        service = build("admin", "directory_v1", credentials=self.creds)
+        updated_creds = get_credentials(
+            self.creds,
+            scopes=[
+                "https://www.googleapis.com/auth/admin.directory.group.member.readonly"
+            ],
+        )
```

Comment:
We probably don't need to rescope the creds every time - can we move this into the object init?

We may need to refresh the creds if they are expired (check if the python connector does this)

Reply:
fixed this by passing in the scoped credentials when initializing the object.